source/simulate/run-simulation.md

```diff
@@ -135,12 +134,12 @@ As for the *test case* content, you will need the following information:
 - the name of the corresponding variable in your model for every set of values 
   > `person_salary` values become [salary](https://demo.openfisca.org/legislation/salary) values in `OpenFisca-Country-Template` model
 - the period and entity for every set of values
-  > `person_salary` and `age` belong to *Person* entity  
+  > `person_salary` and `person_age` belong to *Person* entity  
   > the reference period isn't in the CSV file but it might, for example, come from the CSV creation time and be identical for the whole data set.
 
 
 
-#### Application: calculate a population's income tax from a CSV file
+#### Application with persons entity: calculate a population's income tax from a CSV file
```

Comment:
I don't understand the addition, what do you mean by _with persons entity_? The information is already below: _calculate `income_tax` for all persons_.

Reply:
Maybe just add a link to https://openfisca.org/doc/coding-the-legislation/50_entities.html

Reply:
Shortened to `Application: calculate persons income tax from a CSV file`.
> This removes `population` concept from the title but it seems ok as it isn't defined in the key concepts section yet.